### ISSUE TITLE
Technical checks should verify that every set contains at least 2 sites

### ISSUE THREAD

**Issue by cfredric** [GoogleChrome/related-website-sets#410]
A set with 1 site (just the primary) is pointless, and does not need to be on this list.

**Comment by Meinne1** [comment]
https://github.com/GoogleChrome/related-website-sets/issues/410#issue-2340870921
[Choose your Microsoft Credential.pdf](https://github.com/user-attachments/files/16043239/Choose.your.Microsoft.Credential.pdf)

**Comment by Esornyl** [comment]
My Problem My Facebook it's not open I need help

**Comment by cervantesjoanna06** [comment]
> A set with 1 site (just the primary) is pointless, and does not need to be on this list.
[](url)

**Comment by BapaBapa745** [comment]
[related-website-sets-Add-CLA-Instructions.zip](https://github.com/user-attachments/files/16616924/related-website-sets-Add-CLA-Instructions.zip)

**Comment by 0619966750** [comment]
Facebook
0657712116
p67715228@gmail.com

**Comment by Aquino55** [comment]
> Un conjunto con 1 sitio (sólo el principal) no tiene sentido y no necesita estar en esta lista.

```c++
Duplicate of #767

**Comment by Almhserii1** [comment]
> A set with 1 site (just the primary) is pointless, and does not need to be on this list.

**Comment by ziyadabdurahaman3-create** [comment]
<img width="398" height="398" alt="Image" src="https://github.com/user-attachments/assets/d4a047c2-f78a-4c6e-b270-685f18b7331e" />

**Comment by saadshomeli562-cmyk** [comment]
[https://github.com/GoogleChrome/related-website-sets/issues/410#issuecomment-3160407026](url)

**Comment by saadshomeli562-cmyk** [comment]
[https://github.com/GoogleChrome/related-website-sets/issues/410#issuecomment-3164053419](url)

**Comment by saadshomeli562-cmyk** [comment]
https://github.com/GoogleChrome/related-website-sets/issues/410#issuecomment-3164054755[](url)

**Comment by saadshomeli562-cmyk** [comment]
https://github.com/GoogleChrome/related-website-sets/issues/410#issuecomment-3164056090

**Comment by sajulimu-rgb** [comment]
Ok dane
